phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/LocalRepository.java

```diff
@@ -71,20 +72,21 @@ public RepositoryDTO fetchInstallableApplications() {
 
         final File i18nDirectory = new File(repositoryDirectory, "i18n");
         if (i18nDirectory.exists()) {
-            final File[] translationFiles = i18nDirectory.listFiles((dir, name) -> name.endsWith(".json"));
-            Set<TranslationDTO> translations = new HashSet<>();
-            for (File translation : translationFiles) {
+            final File[] translationFiles = i18nDirectory
+                    .listFiles((dir, name) -> name.endsWith(Locale.getDefault().getLanguage() + ".properties"));
+            Properties mergedProperties = new Properties();
+            for (File translationFile : translationFiles) {
                 try {
-                    final String language = translation.getName();
-                    final String content = new String(Files.readAllBytes(translation.toPath()));
-                    final TranslationDTO.Builder translationDTOBuilder = new TranslationDTO.Builder()
-                            .withLanguage(language).withJson(content);
-                    translations.add(translationDTOBuilder.build());
+                    Properties langProperties = new Properties();
+                    langProperties.load(new FileInputStream(translationFile));
+                    mergedProperties.putAll(langProperties);
```

Comment:
How are overrides handled in this case?
Let's say a key is used in multiple property files, which one is then taken in the end?